dev/how-to/get-started/data-migration.md

```diff
@@ -0,0 +1,649 @@
+---
+title: TiDB DM (Data Migration) Tutorial
+summary: Learn the basics of the TiDB DM (Data Migration) platform, to migrate a simple sharded schema from MySQL to TiDB.
+category: how-to
+---
+
+# TiDB DM (Data Migration) Tutorial
+
+TiDB DM (Data Migration) is a platform that supports migrating large, complex, production data sets from MySQL or MariaDB to TiDB.
+
+DM supports creating and importing an initial dump of data, as well as keeping data synchronized during migration by reading and applying binary logs from the source data store. DM can migrate sharded topologies from in-production databases by merging tables from multiple separate upstream MySQL/MariaDB instances/clusters. In addition to its use for migrations, DM is often used on an ongoing basis by existing MySQL or MariaDB users who deploy a TiDB cluster as a slave, to either provide improved horizontal scalability or run real-time analytical workloads on TiDB without needing to manage an ETL pipeline. 
+
+In this tutorial, we'll see how to migrate a sharded table from multiple upstream MySQL instances. We'll do this a couple of different ways. First, we'll merge several tables/shards that do not conflict; that is, they're partitioned using a scheme that does not result in conflicting unique key values. Then, we'll merge several tables that **do** have conflicting unique key values.
+
+This tutorial assumes you're using a new, clean CentOS 7 instance. You can virtualize locally (using VMware, VirtualBox, etc.), or deploy a small cloud VM on your favorite provider. You'll have the best luck if you have at least 1GB of memory, since we're going to run quite a few services.
+
+> Warning: The methodology used to deploy TiDB in this tutorial should **not** be used to deploy TiDB in a production or development setting.
+
+### Architecture
+
+![TiDB DM architecture](/media/dm-architecture.png)
+
+The TiDB DM (Data Migration) platform consists of 3 components: dm-master, dm-worker, and dmctl.
```

Comment:
This is the capitalization style determined by the tools team. Please also update other occurrences in this PR: DM-master, DM-worker, dmctl.

```suggestion
The TiDB DM (Data Migration) platform consists of 3 components: DM-master, DM-worker, and dmctl.
```